.github/workflows/check_main_script.yaml

```
@@ -31,5 +37,16 @@ jobs:
         run: pip install flake8 flake8-docstrings flake8-import-order
       - name: Check main script (bin) by flake8
         run: flake8 --max-line-length=90 --show-source --statistics truckersmp-cli
+        if: always()
       - name: Check main script (module) by flake8
         run: flake8 --max-line-length=90 --show-source --statistics truckersmp_cli/__init__.py
+        if: always()
+      - name: Check downloads script (module) by flake8
+        run: flake8 --max-line-length=90 --show-source --statistics truckersmp_cli/downloads.py
+        if: always()
+      - name: Check utils script (module) by flake8
+        run: flake8 --max-line-length=90 --show-source --statistics truckersmp_cli/utils.py
+        if: always()
+      - name: Check variables script (module) by flake8
+        run: flake8 --max-line-length=90 --show-source --statistics truckersmp_cli/variables.py
+        if: always()
```

`flake8` can specify multiple files.

Currently (in master branch) we have 2 steps for running `flake8` (1 step for `truckersmp-cli`, 1 step for `__init__.py`) but now I think we can check all the scripts in 1 step:

```suggestion
      - name: Check Python scripts by flake8
        run: flake8 --max-line-length=90 --show-source --statistics truckersmp-cli truckersmp_cli/*.py
```

When `flake8` fails, we can see the error details (errored file, location, and error code/error description).